UnityProject/Assets/Scripts/Items/Others/MixingBowl.cs

```diff
@@ -5,6 +5,8 @@ public class MixingBowl : NetworkBehaviour, IServerInventoryMove
 	public ItemSlot currentSlot;
 	public void OnInventoryMoveServer(InventoryMove info)
 	{
+		if (this.gameObject != info.MovedObject.gameObject) return;
```

Comment:
You're adding these checks in places, but i'm bit confused as to how this could ever be true?

Theres a bug somewhere is this script is being called for the object it's not on?

Reply:
that's the changes, inventory move for the object been moved and its contained items

Reply:
Surely a new interface would be better for that? like OnInventoryContentsMoveServer

Reply:
it just makes more sense it being one interface

Reply:
Ok